.vscode/tasks.json

```diff
@@ -223,6 +264,58 @@
       }
     },
     // ****************
+    // Kernel (python)
+    // ****************
+    // Test
+    {
+      "label": "test (Semantic-Kernel-Python)",
+      "command": "poetry",
+      "type": "shell",
+      "args": ["run", "pytest", "tests/unit"],
+      "problemMatcher": "$msCompile",
+      "group": "test",
+      "presentation": {
+        "reveal": "always",
+        "panel": "shared",
+        "group": "PR-Validate"
+      },
+      "options": {
+        "cwd": "${workspaceFolder}/python"
+      }
+    },
+    {
+      "label": "test (Semantic-Kernel-Python Integration)",
+      "command": "poetry",
+      "type": "shell",
+      "args": ["run", "pytest", "tests/integration","-k","${input:filter}",],
```

Comment:
I use Prettier, and on file save it automatically removes commas after last array item. GitHub also highlights this comma:
![image](https://github.com/microsoft/semantic-kernel/assets/13853051/f881c593-e196-40de-995c-ce572a0b9abf)

I'm wondering what formatter you use and should we setup some rules to use the same formatter for JSON files?

Reply:
Yeah I use prettier too -- sometimes it's flaky for me though. Working now.